src/HexUtils.js

```diff
@@ -40,6 +40,15 @@ class HexUtils {
     return HexUtils.add(hex, HexUtils.direction(direction));
   }
 
+  static neighbours(hex) {
```

Comment:
This method returns hex object that are not necessarily rendered on a grid, which is fine. Just thinking of the different use cases... :P